### PR TITLE
Update sample - Display points using feature reduction 

### PR DESCRIPTION
**Pull request by prupani-7**
## Description
PR to update the sample to use composable mapview from the toolkit
Use "arcgisToolkitVersion = '200.4.0-1'" for MapView Identofy operations
Removed the `show Loading Dialog` when Map is loaded from prev sample implementation with removal of init block. It was added just as an enhancement to the sample. 
Rename the png file, for checker to pass and also sample viewer will show the first image as display for the app

## Links and Data

Sample Epic: `runtime/kotlin/issues/3363`

## What To Review
Review updated files

## How to Test

Run the sample on the sample viewer or the repo.

### CODE REVIEW

**Review by shubham7109 — APPROVED** [review]

File: display-points-using-clustering-feature-reduction/src/main/java/com/esri/arcgismaps/sample/displaypointsusingclusteringfeaturereduction/screens/MainScreen.kt [hunk 1]
```diff
@@ -62,12 +67,17 @@ fun MainScreen(sampleName: String, application: Application) {
...
                         .fillMaxSize()
                         .weight(1f),
-                    mapViewModel = mapViewModel
+                    arcGISMap = mapViewModel.map,
+                    viewpointOperation = MapViewpointOperation.Set(
+                        viewpoint = Viewpoint(39.8, -98.6, 10e7)
+                    ),
+                    mapViewProxy = mapViewModel.mapViewProxy,
+                    onSingleTapConfirmed = mapViewModel::identify,
+                    onPan = { mapViewModel.dismissBottomSheet() }
```
Comment (XuemingRocks) [inline display-points-using-clustering-feature-reduction/src/main/java/com/esri/arcgismaps/sample/displaypointsusingclusteringfeaturereduction/screens/MainScreen.kt:80]: Maybe the following gesture events should also be overridden like the `onPan` event to dismiss the bottom sheet?

- `onDoubleTap`
- `onLongPress`
- `onTwoPointerTap`

**Review by XuemingRocks — APPROVED** [review]
Overall looks good. Just a minor suggestion.

File: display-points-using-clustering-feature-reduction/src/main/java/com/esri/arcgismaps/sample/displaypointsusingclusteringfeaturereduction/screens/MainScreen.kt (hunk 1 above)
Comment (prupani-7) [reply display-points-using-clustering-feature-reduction/src/main/java/com/esri/arcgismaps/sample/displaypointsusingclusteringfeaturereduction/screens/MainScreen.kt:80]: @XuemingRocks I appreciate your feedback, it has prompted some thoughtful consideration on my part.

During the implementation of this sample during the past release, there was a consideration regarding the interaction between the bottom sheet and map usage. It was suggested that allowing the dismissal of the bottom sheet upon initiating map interactions, such as panning to explore and identify additional features, would enhance visibility and prioritize the map's centrality.

Considering this, modifying other gestures to facilitate the dismissal of the bottom sheet would not notably improve the current functionality or user experience. There is also negligible likelihood of users using those specific events to dismiss the bottom sheet, as there is already a close button available on the bottom sheet for the same.

Comment (XuemingRocks) [reply display-points-using-clustering-feature-reduction/src/main/java/com/esri/arcgismaps/sample/displaypointsusingclusteringfeaturereduction/screens/MainScreen.kt:80]: Fair enough

**Comment by prupani-7** [comment]
Thanks for the reviews, @shubham7109 and @XuemingRocks ..Merging

**Pull request merged**